upgrading/upgrade-running-applications/1_deploy.md

```diff
@@ -52,5 +52,5 @@ daml script --dar forum-0.1.0.dar --script-name Init:initialize --ledger-host lo
 ```{{execute T2}}
 
 Nice, your social network is running and users are happily posting and commenting in the forum
-(look at the `../forum/Init.daml`{{open}} module in the forum source code to see what). In the next
+(look at the `../forum/daml/Init.daml`{{open}} module in the forum source code to see what). In the next
```

Comment:
did this work before?

Reply:
It probably didn't as the file path wasn't right